django_mongodb/compiler.py

```diff
@@ -276,6 +425,47 @@ def get_lookup_pipeline(self):
             result += self.query.alias_map[alias].as_mql(self, self.connection)
         return result
 
+    def _get_aggregate_expressions(self, expr):
+        stack = [expr]
+        while stack:
+            expr = stack.pop()
+            if isinstance(expr, Aggregate):
+                yield expr
+            elif hasattr(expr, "get_source_expressions"):
+                stack.extend(expr.get_source_expressions())
+
+    def get_aggregation_pipeline(self):
+        return self._group_pipeline
```

Comment:
Does it need to be a private variable with a `get_` method? Could it be `self.aggregation_pipeline` like other variables in `SQLCompiler.__init__()`?